Scripts/SL-SupportHelpers.lua

```diff
@@ -47,6 +47,7 @@
 -- use StepManiaVersionIsSupported() to check if Simply Love supports the version of SM5 in use
 
 StepManiaVersionIsSupported = function()
+	if true then return true end
```

Comment:
lol please remove, though I appreciate the sentiment

Reply:
Oh no I wasn't supposed to add this to the commit (yes I'm using 5.3 to develop this)